proto/replicationdata.proto

```diff
@@ -25,7 +25,9 @@ package replicationdata;
 // flavor-specific command and parsed into a Position and fields.
 message Status {
   string position = 1;
-  bool io_thread_running = 2;
+  // IoThreadRunningState should be used instead of IoThreadRunning and correctly represents all three valid states from
+  // IO_Thread_Running returned from calling SHOW REPLICA STATUS.
+  bool io_thread_running = 2 [deprecated = true];
```

Comment:
What's the canonical way of doing this? What I have seen before:
```
reserved = 2
```

Reply:
From the protobuf docs: 
```
deprecated (field option): If set to true, indicates that the field is deprecated and should not be used by new code. In most languages this has no actual effect. In Java, this becomes a @Deprecated annotation. In the future, other language-specific code generators may generate deprecation annotations on the field's accessors, which will in turn cause a warning to be emitted when compiling code which attempts to use the field. If the field is not used by anyone and you want to prevent new users from using it, consider replacing the field declaration with a reserved statement.
```